environment.yml

```diff
@@ -9,3 +9,8 @@ dependencies:
   # TODO nf-core: Add required software dependencies here
   - fastqc=0.11.8
   - multiqc=1.7
+  - r-base=3.6.1
+  - conda-forge::r-data.table=1.12.4
```

Comment:
Is data.table used anywhere? Seem not able to locate it, is there any reason why this is also put in the software dependencies?

Reply:
I removed this data.table dependency from environment.yml and push another container on docker (yuukiiwa/nanornabam:1.0).